api/app/controllers/spree/api/v1/credit_cards_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Spree
+  module Api
+    module V1
+      class CreditCardsController < Spree::Api::BaseController
+        before_action :user
+
+        def index
+          @credit_cards = user
+            .credit_cards
+            .accessible_by(current_ability, :read)
+            .with_payment_profile
+            .ransack(params[:q]).result.page(params[:page]).per(params[:per_page])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.